doc/awsrole.md

```diff
@@ -34,3 +34,28 @@ Options:
   - rolename: name of role to attach policy to
   - policyARN: arn of policy to attach to role
 
+### create-sa
+Create a new service account 
+```
+  gen3 awsrole create-sa <serviceaccount>
+```
+Options:
+  - serviceaccount: name of service account
+
+### create-assumerole
+Create a new role for service account with assumerole policy attached 
+```
+    gen3 awsrole create-assumerole <serviceaccount>
+```
+Options:
+  - serviceaccount: name of service account
+  
+### annotate-sa
+Annotate service account with Role
+```
```

Comment:
Put a link out to the AWS documentation on the EKS-IAM integration someplace ...

Reply:
Is something that you are looking for? I made a doc when I first started work on this. Probably needs some changes. 😅